OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -89,9 +91,10 @@ public class ProductionQueue : IResolveOrder, ITick, ITechTreeElement, INotifyOw
 		[Sync] public int CurrentSlowdown { get { return QueueLength == 0 ? 0 : queue[0].Slowdown; } }
 		[Sync] public bool CurrentPaused { get { return QueueLength != 0 && queue[0].Paused; } }
 		[Sync] public bool CurrentDone { get { return QueueLength != 0 && queue[0].Done; } }
-		[Sync] public bool Enabled { get; private set; }
+		[Sync] public bool Enabled { get; protected set; }
 
 		public string Faction { get; private set; }
+		[Sync] public bool ValidFaction { get; private set; }
```

Comment:
I don't understand why you folks so dislike the "Is" prefix on boolean properties :/